### DISCUSSION TITLE
Model.exe(executable) doesnt get run in the command-line

### DISCUSSION THREAD

**Discussion by ap766** [OpenModelica/OpenModelica#12251] [Q&A]
So i had actually set up openmodelica before and run the .exe files by including the dll files in the same directory and it gave me this result that it is succesful

<img width="681" alt="image" src="https://github.com/OpenModelica/OpenModelica/assets/79255079/3845a4a0-4f59-4cba-abc3-a14842076527">

But i had to reinstall everything again bcus had to dual boot my system.
And when i am trying again to execute a newly generated .exe with the dependent files it isnt giving me any output.Can someone please tell me whats missing here?I have the dll files in the same directory and the .exe too..but it isnt showing success like before.At the same time the old .exe files are still showng successful.But i dont get what I am doing wrong this time.

<img width="877" alt="image" src="https://github.com/OpenModelica/OpenModelica/assets/79255079/c5319a9f-c883-4046-97e6-9d03259971fb">

**Comment by ap766** [comment]
I have shown it for different models here .But i am not able to run the .exe of any model whether it the test.exe or batch_rectifier.exe

**Reply by AnHeuermann** [reply]
Run the *.bat file instead.

**Reply by ap766** [reply]
its still not showing anything

**Reply by ap766** [reply]
I moved the .exe and dependent dll files to another directory , now i get this error 
<img width="848" alt="image" src="https://github.com/OpenModelica/OpenModelica/assets/79255079/323758f2-78b5-4edb-8836-f86417d0af32">

**Reply by AnHeuermann** [reply]
You need to copy/move more than the .exe and .dll files. I think also the .bat, .xml and .json files.
In a nutshell don't move the executable. If you want something portable use FMUs.

**Reply by AnHeuermann** [reply]
If you compiled the Model `Modelica.Blocks.Examples.PID_Controller` with OMEdit you can run the simulation by calling the `PID_Controller.bat` file from a cmd terminal:

```
C:\Users\USERNAME\AppData\Local\Temp\OpenModelica\OMEdit\Modelica.Blocks.Examples.PID_Controller>.\PID_Controller.bat

LOG_SUCCESS       | info    | The initialization finished successfully with 3 homotopy steps.
LOG_SUCCESS       | info    | The simulation finished successfully.
```

**Reply by ap766** [reply]
yes i was actually trying this but I am not getting any output for some reason.Even when i include all the files together (xml,dll,etc)

**Reply by ap766** [reply]
<img width="475" alt="image" src="https://github.com/OpenModelica/OpenModelica/assets/79255079/305e5400-4b3d-4358-8e5f-30fbecf0b53a">
I think path also has been set up correctly

**Reply by AnHeuermann** [reply]
Try to update your OpenModelica. The latest version is v1.22.3 or v1.24.0-def. If that's not helping maybe check if your anti virus or Microsoft Defender blocked something.

**Reply by ap766** [reply]
Yes,its not an issue with the version because I tried with  v1.22.3 prior to this .Will check out if something is blocked though.Thanks a lot.
